src/tasks.js

```diff
@@ -1,19 +1,33 @@
 const generateId = () => (Math.random() + 1).toString(36).substring(7);
 
-export const Task = (description) => {
-  const completedStatus = false;
-  const id = generateId();
+const generateIndex = () => {
+  if (localStorage.getItem('tasksArray') === null) {
+    return 0;
+  }
+  const storedTasksArray = JSON.parse(localStorage.getItem('tasksArray'));
+  return storedTasksArray.length;
+};
+
+export const Task = (taskData) => {
+  const {
+    description, completedStatus = false, id = generateId(), index = generateIndex(),
+  } = taskData;
```

Comment:
- [ ] Don't you think `Math.random() + 1).toString(36).substring(7)` this method will not give you unique id every time. I am sharing all the comments [here](https://stackoverflow.com/a/8084248/15599817) that this is not good for generating unique id's and instead for simple we can use counter to update the index starting from `1` here..

Reply:
Hi Dipeshtwis, I have removed all commented code from my project 👍 
 In the comment, you linked it said 50, 000 strings, the project is only for few tasks not up to a thousand. Using this method is more efficient in my opinion and I would like to keep it for this small project
However, I will keep the suggestion in mind and use a library if I really need unique numbers ❤️

Reply:
Hi @bimbolabuari,

I know this project is not so big and you have done a good job finding something new to implement but make sure while dealing with large data always find something which last long like `UUID`, I would suggest. :wink: